README.md

```diff
@@ -52,16 +72,25 @@ Authors
 Jordi Boggiano - <j.boggiano@seld.be> - <http://twitter.com/seldaek> - <http://seld.be><br />
 Nils Adermann - <naderman@naderman.de> - <http://twitter.com/naderman> - <http://www.naderman.de><br />
 
-See also the list of [contributors](https://github.com/composer/satis/contributors) who participated in this project.
+See also the list of [contributors][] who participated in this project.
```

Comment:
Yeah it works on Github, but according to https://daringfireball.net/projects/markdown/syntax#link it should be `[label][]`.